news/utils.py

```diff
@@ -485,8 +498,11 @@ def get_best_language(languages):
     if not languages:
         return None
 
+    # try again with 2 letter languages
+    languages_2l = [lang[:2] for lang in languages]
```

Comment:
Confirmed. So far we only deal with a small list of languages, and none are 3 letter.